examples/01_plotting/plot_demo_plotting.py

```diff
@@ -16,7 +16,7 @@
 from nilearn import datasets
 
 # haxby dataset to have EPI images and masks
-haxby_dataset = datasets.fetch_haxby(n_subjects=1)
+haxby_dataset = datasets.fetch_haxby(subjects=[1])
```

Comment:
Let's try to always use subject 2, to limit download (and probably simply always use the default, to simplify the code).